.github/workflows/docs.yml

```
@@ -38,5 +38,5 @@ jobs:
       id-token: write
 
     with:
-      bazel-target: "//docs:github_pages__release"
+      bazel-target: "//docs:incremental_release -- --github_user=nicu1989 --github_repo=docs-as-code"
```

```suggestion
      bazel-target: "//docs:incremental_release -- --github_user=$GITHUB_REPOSITORY_OWNER --github_repo=$GITHUB_REPOSITORY"
```

But I'm not sure those are available within `with`. Maybe it's:

```suggestion
      bazel-target: "//docs:incremental_release -- --github_user=${{env.GITHUB_REPOSITORY_OWNER} --env.github_repo=${{GITHUB_REPOSITORY}"
```